Source/ACE/Entity/GameData.cs

```diff
@@ -1,63 +1,57 @@
-﻿using ACE.Entity.Enum;
-using ACE.Network;
-using ACE.Network.Enum;
-using ACE.Network.GameMessages;
-using ACE.Network.GameMessages.Messages;
-using ACE.Network.Managers;
-using System.IO;
+﻿using ACE.Network.Enum;
 
 namespace ACE.Entity
 {
+    /// <summary>
+    /// GameData - which of these items get written out is controled by DescriptionFlags
+    /// </summary>
```

Comment:
// TODO: replace with exiting data in AceObject rather than copying it to this object